Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/AssetBrowserEntityInspectorWidget.h

```diff
@@ -26,11 +33,32 @@ namespace AzToolsFramework
         public:
             explicit AssetBrowserEntityInspectorWidget(QWidget *parent = nullptr);
             ~AssetBrowserEntityInspectorWidget();
-        private:
+
+            void InitializeDatabase();
+
+            //////////////////////////////////////////////////////////////////////////
+            // AssetBrowserPreviewRequestBus
+            //////////////////////////////////////////////////////////////////////////
             void PreviewAsset(const AzToolsFramework::AssetBrowser::AssetBrowserEntry* selectedEntry) override;
+            void ClearPreview() override;
 
+            void PopulateSourceDependencies(const SourceAssetBrowserEntry* sourceEntry, AZStd::vector<const ProductAssetBrowserEntry*> productList);
+            bool PopulateProductDependencies(const ProductAssetBrowserEntry* productEntry);
+            void CreateSourceDependencyTree(AZStd::set<AZ::Uuid> sourceUuids, bool isOutgoing);
+            void CreateProductDependencyTree(AZStd::set<AZ::Data::AssetId> dependencyUuids, bool isOutgoing);
+            void AddAssetBrowserEntryToTree(const AssetBrowserEntry* entry, QTreeWidgetItem* headerItem);
         private:
-            PreviewerFrame* m_previewerFrame = nullptr;
+            AZStd::shared_ptr<AssetDatabase::AssetDatabaseConnection> m_databaseConnection;
+            bool m_dbReady = false;
+            QLabel* m_previewImage = nullptr;
+            QStackedLayout* m_layoutSwitcher = nullptr;
+            QWidget* m_emptyLayoutWidget = nullptr;
+            QWidget* m_populatedLayoutWidget = nullptr;
+            QFormLayout* m_assetDetailLayout = nullptr;
+            QWidget* m_assetDetailWidget = nullptr;
+            QTreeWidget* m_dependentProducts = nullptr;
+            AzQtComponents::Card* m_dependentAssetsCard = nullptr;
+            QFont m_headerFont;
```

Comment:
Some suggestions:
1. `m_dbReady` can be renamed to `m_databaseReady`
2. I think it is more readable and easier to maintain if they can be grouped either by Qt types or by their functionalities.

Reply:
Changed m_dbReady to m_databaseReady, and grouped the variables by their functionalities